components/TextControls/src/Theming/MDCFilledTextField+MaterialTheming.m

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCFilledTextField+MaterialTheming.h"
+
+#import <Foundation/Foundation.h>
+
+#import "../../src/private/MDCBaseTextField+MDCTextControl.h"
+#import "../../src/private/MDCTextControl.h"
+#import "../../src/private/MDCTextControlStyleFilled.h"
+
+@implementation MDCFilledTextField (MaterialTheming)
+
+- (void)applyThemeWithScheme:(nonnull id<MDCContainerScheming>)containerScheme {
+  [self applyTypographyScheme:[self typographySchemeWithContainerScheme:containerScheme]];
+  [self applyDefaultColorScheme:[self colorSchemeWithContainerScheme:containerScheme]];
+}
+
+- (void)applyErrorThemeWithScheme:(nonnull id<MDCContainerScheming>)containerScheme {
+  [self applyTypographyScheme:[self typographySchemeWithContainerScheme:containerScheme]];
+  [self applyErrorColorScheme:[self colorSchemeWithContainerScheme:containerScheme]];
+}
+
+- (id<MDCColorScheming>)colorSchemeWithContainerScheme:
+    (nonnull id<MDCContainerScheming>)containerScheme {
```

Comment:
Why is this needed? Isn't color and typography `nonnull`?

Reply:
I think I wrote this before that change was made haha

Reply:
Changed it.